PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,3 +30,4 @@ As a Codecademy Contributor, by submitting any "Asset" to Codecademy via GitHub,
 **_Asset License._** Subject to the terms of this Contributor Agreement, Codecademy grants you a perpetual, royalty-free, nonexclusive, nontransferable, worldwide right to use the Asset provided you may not use the Asset in any way that directly competes with Codecademy.
 
 **_Contributor Spotlights._** You agree Codecademy may use your information including your personal information (e.g., username, date of account, number of contributions) and information related to the Asset (e.g., topic, type etc.), that you provide to Codecademy (i.e., via your account profile page, contributor submission form, or otherwise), to market and advertise Codecademy’s Contributor program and related Contributor spotlight pages all subject to Codecademy’s Privacy Policy.
+-->
```

Comment:
not sure if we actually want this to be visible on each PR or if a comment is sufficient